apps/sms/services/js/message_manager.js

```diff
@@ -12,9 +12,22 @@
 
 /*exported MessageManager */
 
+/**
+ * MessageManager is responsible for
+ *   1. handling message events
+ *   2. handling message threads and their read status
+ *   3. sending SMS and MMS messages
+ *
+ * @module MessageManager
+ */
 'use strict';
 (function(exports) {
 var MessageManager = {
+
+  /**
+   * Standard initialization function that needs to be run
```

Comment:
__nit:__ Usually description should tell what method/function does :) How about ```Initializes MessageManager and subscribes for required {@link http://mdn.io/MozMobileMessageManager} events```?